### DISCUSSION TITLE
Is papis for notes and quotes?

### DISCUSSION THREAD

**Discussion by gelerum** [papis/papis#897] [Q&A] [answered]
Is storing my thoughts and quotes from papers purpose of papis?

**Comment by tuurep** [comment, marked as answer]
It can certainly be, at least that's one of my most important use cases for it.

So I have a paper that I can open with `papis open foobar`, and an associated markdown note with `papis edit -n foobar`

I didn't like the default `.tex` extension for the notes so I set that as `.md` like

`~/.config/papis/config`

```
[settings]

notes-name = note.md
```

(Replace `note` with a template if preferred)

**Reply by gelerum** [reply]
Do you somehow add information about page? I think i can do this only with latex

**Reply by tuurep** [reply]
I'm not sure what exactly you mean - like doing citations with page numbers?

Actually makes me think that markdown isn't the right tool for it, yeah

I've mostly written freeform text or quoted something with markdown blockquotes and copypasting.

**Reply by gelerum** [reply]
Yeah, i meant that. Thank you. I think for university i need to save a part of a text along with its page.
